[2022-07-19 09:43:08,811] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T09:42:00+00:00 [queued]>
[2022-07-19 09:43:08,820] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T09:42:00+00:00 [queued]>
[2022-07-19 09:43:08,820] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 09:43:08,821] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-19 09:43:08,821] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 09:43:08,839] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): respond> on 2022-07-19 09:42:00+00:00
[2022-07-19 09:43:08,844] {standard_task_runner.py:52} INFO - Started process 5416 to run task
[2022-07-19 09:43:08,847] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'simple_dag', 'respond', 'scheduled__2022-07-19T09:42:00+00:00', '--job-id', '268', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpywx136xb', '--error-file', '/tmp/tmpka22qwkk']
[2022-07-19 09:43:08,848] {standard_task_runner.py:80} INFO - Job 268: Subtask respond
[2022-07-19 09:43:08,904] {task_command.py:370} INFO - Running <TaskInstance: simple_dag.respond scheduled__2022-07-19T09:42:00+00:00 [running]> on host 02ffb48252da
[2022-07-19 09:43:08,981] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-07-19T09:42:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-19T09:42:00+00:00
[2022-07-19 09:43:15,171] {logging_mixin.py:115} INFO - (2724, 1)
[2022-07-19 09:43:15,179] {python.py:173} INFO - Done. Returned value was: Greet Responded Again
[2022-07-19 09:43:15,297] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=respond, execution_date=20220719T094200, start_date=20220719T094308, end_date=20220719T094315
[2022-07-19 09:43:15,340] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-19 09:43:15,368] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
